src/pages/__tests__/Contact.test.tsx

```diff
@@ -17,3 +31,102 @@ describe("Test Contact Page", () => {
         expect(wrapper.find("ContactForm").length).toBe(1);
     });
 });
+
+// test sendMessage function
+describe("Test sendMessage function", () => {
+    const setIsError = jest.fn();
```

Comment:
If you will reuse a mock then you should jest.resetAllMocks() before each test. Another option is to add to the jest configuration file to reset all mocks. This resets the mock to a clear state so tests can't interfere with each other.